tdms/include/arrays.h

```diff
@@ -172,6 +196,16 @@ class DispersiveMultiLayer{
 
 public:
   explicit DispersiveMultiLayer(const mxArray *ptr);
+
+  /**
+   * @brief Determines whether the (background) medium is dispersive
+   *
+   * @param K_tot Number of Yee cells in the z-direction (number of entries in this->gamma)
+   * @param near_zero_tolerance Tolerance for non-zero gamma (attenuation) values
+   * @return true Background is dispersive
+   * @return false Background is not dispersive
+   */
+  bool is_dispersive(int K_tot, double near_zero_tolerance=1e-15);
```

Comment:
So we can have a non-dispersive dispersive multilayer? (`DispersiveMultiLayer::is_dispersive == false`)

Reply:
Apparently so - there's a corresponding call in `iterator.cpp`. I don't have the same feel for everything _in_ `iterator.cpp` yet like Tom did, so I'm guessing its an oddity for us to be in this case rather than common.

There's also another [`is_dispersive`](https://github.com/UCL/TDMS/blob/89fce4c9acc7e4db95d4e533dfc6e84c01a7acc0/tdms/src/iterator.cpp#L873-L875) function for the background material (which doesn't have its own class) too.